src/Tribe/Importer/Options.php

```diff
@@ -124,4 +137,54 @@ public static function get_possible_stati() {
 
 		return apply_filters( 'tribe_import_possible_stati', $stati );
 	}
+
+	/**
+	 *  Return Encoding Status
+	 *
+	 * @param string $type importer type
+	 *
+	 * @return mixed|void
+	 */
+	public static function get_default_encoding_status( $type = 'csv' ) {
+
+		$options = self::getOption( 'imported_encoding_status', array( $type => 'encode' ) );
+
+		if ( ! isset( $options[ $type ] ) ) {
+			$options[ $type ] = 'encode';
+		}
+
+		if ( ! isset( $options[ $type ] ) ) {
```

Comment:
Don't we use this same condition right above here (line 152) and doesn't that mean this second condition will never be fulfilled (because if `$options[$type]` wasn't set, it will be by this point) ... maybe we need to combine them?